Dockerfile

```
@@ -0,0 +1,5 @@
+
+FROM openjdk:17-jdk-slim
+COPY target/Online-Book-Store-0.0.1-SNAPSHOT.jar online-book-store.jar
```

You can use wildcard for more flexibility
```suggestion
COPY target/*.jar online-book-store.jar
```